packages/sdk/src/interaction/interaction.js

```diff
@@ -63,10 +73,11 @@ const IX = `{
   "accounts":{},
   "params":{},
   "arguments":{},
+  "metadata":{},
   "message": {
     "cadence":null,
     "refBlock":null,
-    "computLimit":null,
+    "computeLimit":null,
```

Comment:
updated what seems to be a typo

Reply:
:+1:

Reply:
cc: @JeffreyDoyle

Reply:
I believe this is fine!

The `limit` builder sets a limit on `computeLimit`. So the typo'd field on the interaction object is likely never being used.